### PR TITLE
Add Python 3.3 support

### PR DESCRIPTION
**Pull request by DemonVex**

### CODE REVIEW

File: multisite/models.py [hunk 1]
```diff
@@ -74,7 +78,7 @@ def _expand_netloc(cls, host, port=None):
             bits = host.split('.')
 
         result = []
-        for i in xrange(0, (len(bits) + 1)):
+        for i in range(0, (len(bits) + 1)):
```
Comment (jordiecometrica) [inline multisite/models.py]: Doesn't this change behaviour in Python 2.7? Is there a way to fix this without changing behaviour?

File: multisite/threadlocals.py [hunk 2]
```diff
@@ -41,7 +43,7 @@ def __init__(self, default=None, *args, **kwargs):
         ``default``, if specified, determines the default SITE_ID,
         if that is unset.
         """
-        if default is not None and not isinstance(default, (int, long)):
```
Comment (jordiecometrica) [inline multisite/threadlocals.py]: These and the following changes will not work the same way on Python 2.7 either.

File: multisite/models.py (hunk 1 above)
Comment (DemonVex) [inline multisite/models.py]: "range" was imported from builtins module. More info http://python-future.org/compatible_idioms.html#xrange

File: multisite/threadlocals.py (hunk 2 above)
Comment (DemonVex) [inline multisite/threadlocals.py]: http://python-future.org/compatible_idioms.html#long-integers

**Comment by DemonVex** [comment]
Sorry, I forgot about dependency from the "future" library. I've fixed that. It use "six" now.

**Comment by jordiecometrica** [comment]
Okay, thanks. I tested your changes on Python 2 and there appears to be no breakage.

**Pull request merged**